frontend/src/fixtures/commonsFixtures.js

```diff
@@ -26,7 +28,9 @@ const commonsFixtures = {
             "milkPrice": 10,
             "degradationRate": .5,
             "showLeaderboard": true,
+            "capacityPerUser": 11,
             "carryingCapacity": 123,
+            "effectiveCapacity": 123,
```

Comment:
If the totalPlayers are 50 and the capacityPerUser is 11, shouldn't the effectiveCapacity be 550?